examples/with-emotion/pages/_document.js

```diff
@@ -1,28 +1,24 @@
 import Document, { Head, Main, NextScript } from 'next/document'
+import { renderToString } from 'react-dom/server'
 import { extractCritical } from 'emotion-server'
 
 export default class MyDocument extends Document {
   static async getInitialProps(ctx) {
     const initialProps = await Document.getInitialProps(ctx)
```

Comment:
Calling this calls `renderToString`. Next.js is in charge of React rendering and it shouldn't be re-passed into `renderToString` as that'll slow down your app for no good reason (it doesn't have any effect).